docs/UserGuide.md

```diff
@@ -19,11 +19,12 @@
 2. Execute it on the OS terminal. <br>
 Usage: `java -jar RepoSense.jar -config CSV_CONFIG_FILE_PATH [-output OUTPUT_DIRECTORY] [-since DD/MM/YYYY] [-until DD/MM/YYYY]`
 3. The dashboard will be generated in the designated OUTPUT_DIRECTORY, or current working directory otherwise, upon successful compilation as `index.html`.
-4. Start a webserver in the generated report directory to server the files
+4. To visualize the report, start a webserver in the generated report directory to server the files
```

Comment:
Doesn't really explain well. Try to be more elaborative, e.g. why is it needed to start a webserver?

Alternatively, I think we should just scrape the entire web-server thing for local preview. People are unlikely to use it unless you're able to deliver them the cost and benefit. Let's go with using zipped file for local instead.

Reply:
roger that, removing the web server part